Composer/packages/client/src/pages/publish/publishStatusList.tsx

```diff
@@ -54,16 +53,12 @@ function onRenderDetailsHeader(props, defaultRender) {
 }
 
 export const PublishStatusList: React.FC<IStatusListProps> = (props) => {
-  const { items, onItemClick, groups } = props;
-  const [selectIndex, setSelectedIndex] = useState<number>();
+  const { items, onLogClick, onRollbackClick } = props;
   const [currentSort, setSort] = useState({ key: 'PublishDate', descending: true });
   const sortByDate = (ev: React.MouseEvent<HTMLElement>, column: IColumn): void => {
     if (column.isSorted) {
       column.isSortedDescending = !column.isSortedDescending;
-      let newItems: IStatus[] = [];
-      for (const group of groups) {
-        newItems = newItems.concat(items.slice(group.startIndex, group.startIndex + group.count).reverse());
-      }
+      const newItems: IStatus[] = items.reverse();
```

Comment:
Could u explain why we need to reverse the list?

Reply:
Becaust the function is to sort the list. If the original list is sorted by descending, call the function, the list should be sorted by ascending, so I reverse the original list.